src/pages/workspaces/workspace/ShareWorkspaceModal/ShareWorkspaceModal.ts

```diff
@@ -192,14 +75,19 @@ const ShareWorkspaceModal = ({
   });
 
   useLayoutEffect(() => {
-    !!lastAddedEmail && list.current.scrollTo({ top: list.current.scrollHeight, behavior: 'smooth' });
+    !!lastAddedEmail && list?.current?.scrollTo({ top: list?.current?.scrollHeight, behavior: 'smooth' });
```

Comment:
How would `list` be undefined?

I'm not sure how/where `current` is set, so not sure if undefined is legitimate for `current`.

Reply:
Before the initial render, the value is null.
Also, if I don't do the null conditional, I get a type error.